redirects.yaml

```diff
@@ -731,8 +731,8 @@ services/?: "/support"
 services/server/?: "/server/management"
 shipit.*: "https://blog.canonical.com/2011/04/05/shipit-comes-to-an-end/"
 sites/.*: "/blog"
-smart-start(?P<path>.*)/?: "/core/services/guide{path}"
-smartstart(?P<path>.*)/?: "/core/services/guide{path}"
+smart-start(?P<path>.*)/?: "/core/services/guide"
+smartstart(?P<path>.*)/?: "/core/services/guide"
```

Comment:
Have the paths changed significantly? Would it be best to append the path to the redirection, if we have it?

Reply:
As far as I know smartstart page disappeared, and I don't see any other paths apart from smartstart and smartstart/guide